### PR TITLE
feat(governance): protocol upgrade Unlock v13 and PublicLock v14 on Sepolia

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This adds the manifest files for deployments of Unlock v13 and PublicLock v14 on Sepolia, as well as a few fixes in the deployment / upgrade scripts 

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/contracts/src/contracts/PublicLock/PublicLockV14.sol [hunk 1]
```diff
@@ -5347,7 +5347,7 @@ pragma solidity ^0.8.21;
  * Every ERC-721 compliant contract must implement the ERC165 interface.
  * https://eips.ethereum.org/EIPS/eip-721
  */
-contract PublicLockV14 is
+contract PublicLock is
```
Comment (clemsos) [inline packages/contracts/src/contracts/PublicLock/PublicLockV14.sol:5350]: this is actually needed for consistency across version. All contracts are called `Unlock` or `PublicLock`, only the name of the file differs. That is the pattern that is used across the scripts. I changed it before but better to not number the contract name actually

File: governance/scripts/multisig/submitTx.js [hunk 2]
```diff
@@ -5,29 +5,21 @@ const {
   submitTxOldMultisig,
   confirmMultisigTx,
 } = require('./_helpers')
-const { ADDRESS_ZERO } = require('@unlock-protocol/hardhat-helpers')
+const { ADDRESS_ZERO, getNetwork } = require('@unlock-protocol/hardhat-helpers')
 
-const Safe = require('@safe-global/safe-core-sdk').default
-const SafeServiceClient = require('@safe-global/safe-service-client').default
-const EthersAdapter = require('@safe-global/safe-ethers-lib').default
```
Comment (clemsos) [inline governance/scripts/multisig/submitTx.js:12]: these libs ave been deprec by Safe team

File: governance/scripts/upgrade/propose.js [hunk 3]
```diff
@@ -44,7 +44,9 @@ async function main({
       calldata: data,
     },
     signer,
-  })
+  }
+  const transactionId = await submitTx(txArgs)
+  console.log(transactionId)
```
Comment (julien51) [inline governance/scripts/upgrade/propose.js:49]: Maybe show more details?

File: packages/networks/src/networks/sepolia.ts [hunk 4]
```diff
@@ -48,7 +48,7 @@ export const sepolia: NetworkConfig = {
   isTestNetwork: true,
   keyManagerAddress: '0x338b1f296217485bf4df6CE9f93ab4C73F72b57D',
   maxFreeClaimCost: 1000,
-  multisig: '', // SAFE does not support Sepolia as of October 11th 2023
```
Comment (julien51) [inline packages/networks/src/networks/sepolia.ts:51]: yay!

File: packages/contracts/src/contracts/PublicLock/PublicLockV14.sol (hunk 1 above)
Comment (clemsos) [reply packages/contracts/src/contracts/PublicLock/PublicLockV14.sol:5350]: I actually moved this to its own PR as more changes are required (e.g. re packaging the bytecode) https://github.com/unlock-protocol/unlock/pull/13197

**Comment by clemsos** [comment]
UnlockSwapBurner deployed at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68

**Comment by clemsos** [comment]
remaining tasks for sepolia

- deploy UDT
- create UDT/WETH pool
- set UDT in Unlock
- test swap and burn !

**Pull request merged**